packages/html-spec/src/pager/pager.json

```diff
@@ -0,0 +1,152 @@
+[
+    {
+        "component": "pager",
+        "selector": [
+            ".k-pager"
+        ],
+        "states": [
+            {
+                "name": "focus",
+                "selector": [
+                    ":focus",
+                    ".k-focus"
+                ]
+            },
+            {
+                "name": "disabled",
+                "selector": [
+                    ":disabled",
+                    "[disabled]",
+                    ".k-disabled"
+                ]
+            }
+        ],
+        "options": [
+            {
+                "name": "size",
+                "values": [
+                    {
+                        "name": "small",
+                        "selector": [
+                            ".k-pager-sm"
+                        ]
+                    },
+                    {
+                        "name": "medium",
+                        "selector": [
+                            ".k-pager-md"
+                        ]
+                    },
+                    {
+                        "name": "large",
+                        "selector": [
+                            ".k-pager-lg"
+                        ]
+                    }
+                ]
+            }
+        ],
+        "children": [
+            {
+                "name": "pager-nav",
+                "$ref": "button",
+                "selector": [
+                    ".k-pager-nav.k-button"
+                ],
```

Comment:
I'm not sure if we need the .k-button selector here and below

Reply:
I've removed it from the navigation button selector, but for the other buttons it's needed in my opinion otherwise we will be targeting only their container `k-pager-numbers`.

Reply:
If the k-pager-numbers is a container and the button is inside, then the selector should be .k-pager-numbers .k-button

Reply:
You are correct and I left it where it is needed https://github.com/telerik/kendo-themes/pull/4032/files#diff-2a82e99c6c536d38727dbba6094a7916f01be2d93abb746f740ea2bf8dfc111aR71